README.md

```diff
@@ -1 +1,80 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+ 
+  <br/>
+
+  <h3><b>"Hello Microverse"</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#A-program-for-greeting-users)
+  - [🛠 Built With](#built-with-HMTL-CSS)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-Microverse] <a name="A-program-for-new-users"></a>
+
+**[Hello-Microverse]** is a program for new users.
+
+## 🛠 Built With <a name="HTML-CSS"></a>
+
+## Built With
+
+- html
+- css
+
+To get a local copy up and running follow these simple example steps.
+
+
+
+### Setup 
+clone the project or download the files to your local environment.
+
+### Install
+you have run the installation commands.
+
+### Usage 
+use linters to check the errors in your html and style 
+
+
+## 👥 Authors <a name="authors"></a>
+
+- GitHub: [@githubhandle](https://github.com/Roland-Ntwali)
+- Twitter: [@twitterhandle](https://twitter.com/_Ntwali)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/roland-ntwali-11b16617b/)
+
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Have a great time
```

Comment:
- [ ] Good job :+1: but as the previous reviewer requested, Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections are present in yours. But now comparing your readme to this [Microverse Readme template](https://github.com/microverseinc/readme-template), you will see that you are missing some necessary headings like `key features`, `Future Features`, `Contributions`, and ` Getting Started` that describe how a new developer could make use of your project by following some steps.